README.md

```diff
@@ -4,168 +4,61 @@
 
-The following commands will use the `opence-env.yaml` Open-CE environment file from a specific Open-CE release to build all of the Open-CE packages for Python 3.6, 3.7 and 3.8, including only CUDA builds. The commands should be run from within the same directory that contains `local_files`.
-
-```bash
-# Clone a specific Open-CE release from GitHub
-git clone https://github.com/open-ce/open-ce.git --branch open-ce-v1.0.0
-# Clone Open-CE Environments from GitHub
-git clone https://github.com/open-ce/open-ce-environments.git --branch open-ce-v2021.1.0
-# Build packages
-./open-ce/open_ce/open-ce build env --python_versions 3.6,3.7,3.8 --build_types cuda ./open-ce-environments/envs/opence-env.yaml
-```
-
-Builds can also be performed without cloning the environments repo, by specifying the URL of an environment file. For example:
-
-```bash
-# Clone Open-CE from GitHub
-git clone https://github.com/open-ce/open-ce.git
-# Build packages
-./open-ce/open_ce/open-ce build env https://raw.githubusercontent.com/open-ce/open-ce-environments/main/envs/opence-env.yaml
-```
-
-#### Building within a container
-
-Passing the `--container_build` argument to the `open-ce build env` command will create a container image and perform the actual build inside of a container based on that image. This will provide a "clean" environment for the builds and make builds more system independent. It is recommended to build with this option as opposed to running on a bare metal machine. For more information on the `--container_build` option, please see [`doc/README.open_ce_build.md`](doc/README.open_ce_build.md#open-ce-build-env-sub-command).
-
-### Building a Single Feedstock
-
-The `open-ce build feedstock` command can be used to build a single feedstock (which could produce one or more conda packages). The output from running `open-ce build feedstock` will be a local conda channel (by default called `condabuild`). For more details on `open-ce build feedstock`, please see [`doc/README.open_ce_build.md`](doc/README.open_ce_build.md#open-ce-build-feedstock-sub-command).
+## GETTING STARTED
 
-The following commands will build all of the packages within a feedstock named `MY_FEEDSTOCK`.
+### Requirements
 
-```bash
-# Clone Open-CE from GitHub
-git clone https://github.com/open-ce/open-ce.git
-# Clone Open-CE Environments from GitHub
-git clone https://github.com/open-ce/open-ce-environments.git
-# Clone MY_FEEDSTOCK from GitHub
-git clone https://github.com/open-ce/MY_FEEDSTOCK-feedstock.git
-# Build packages
-./open-ce/open_ce/open-ce build feedstock --conda_build_config ./open-ce-environments/envs/conda_build_config.yaml --working_directory ./MY_FEEDSTOCK-feedstock
-```
+* `conda` >= 3.8.3
+  * The conda tool can either be installed through [Anaconda](https://www.anaconda.com/products/individual#Downloads) or [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
 
 ### Installing Packages
```

Comment:
Let's change this sections to ### Community Builders

We can also list MIT:
https://opence.mit.edu/#/